install/iso_creation/cd_or_vm/ks.cfg

```diff
@@ -69,22 +69,62 @@ else
   echo "platform is 'integralstor'"
 fi
 
+
+### Running a for loop on Disks available among one will be selected by user/admin for OS to Install ###
+<<<<<<< 16f4fc60b4242c94fdb1501886ca305d0ab90a72
```

Comment:
You need to properly resolve the git merge conflicts. I'm quite sure this script will not execute successfully with all the git strings in it.